web_src/js/features/common-issue.js

```diff
@@ -2,15 +2,34 @@ import $ from 'jquery';
 import {updateIssuesMeta} from './repo-issue.js';
 
 export function initCommonIssue() {
-  $('.issue-checkbox').on('click', () => {
-    const numChecked = $('.issue-checkbox').children('input:checked').length;
-    if (numChecked > 0) {
+  const checkboxOperate = (e) => {
+    const issuecheckbox = $('.issue-checkbox input');
+    if (e.shiftKey && window.checkboxfirst !== undefined) {
+      for (let i = window.checkboxfirst + 1, j = issuecheckbox.index($(e.currentTarget).find('input')); i < j; i++) {
+        issuecheckbox[i].checked = 1;
+      }
+      delete window.checkboxfirst;
```

Comment:
Please use a local variable for this, inside the `initCommonIssue` function.

Reply:
Is that os?
https://github.com/go-gitea/gitea/pull/20177/commits/25fa1d85920d668ce07c3f8be2cac796a50ddd95

Reply:
That's better. No need to use `window.`